packages/contracts-bedrock/src/L1/OPPrestateUpdater.sol

```diff
@@ -94,26 +95,26 @@ contract OPPrestateUpdater is OPContractsManager {
                 revert PrestateRequired();
             }
 
+            // Get the DisputeGameFactory and existing game implementations
             IDisputeGameFactory dgf =
                 IDisputeGameFactory(_prestateUpdateInputs[i].systemConfigProxy.disputeGameFactory());
-
             IFaultDisputeGame fdg = IFaultDisputeGame(address(getGameImplementation(dgf, GameTypes.CANNON)));
-
             IPermissionedDisputeGame pdg =
                 IPermissionedDisputeGame(address(getGameImplementation(dgf, GameTypes.PERMISSIONED_CANNON)));
 
+            // All chains must have a permissioned game, but not all chains must have a fault dispute game.
+            // Whether a chain has a fault dispute game determines how many AddGameInput objects are needed.
             bool hasFDG = address(fdg) != address(0);
 
             AddGameInput[] memory inputs = new AddGameInput[](hasFDG ? 2 : 1);
             AddGameInput memory pdgInput;
             AddGameInput memory fdgInput;
 
-            // Get the current game implementation to copy parameters from
-            uint256 initBond = dgf.initBonds(GameTypes.PERMISSIONED_CANNON);
 
-            // Get the existing game parameters
+            // Get the existing game parameters and init bond for the permissioned game
             IFaultDisputeGame.GameConstructorParams memory pdgParams =
                 getGameConstructorParams(IFaultDisputeGame(address(pdg)));
+            uint256 initBond = dgf.initBonds(GameTypes.PERMISSIONED_CANNON);
```

Comment:
the only logical change made in this PR s moving the `initBond` reads down below the params reads. 

Before this it was kind of floating up above. I think this is cleaner and more readable.

I did the same below as well.

Reply:
Makes sense